docs/install-upgrade.md

```diff
@@ -20,6 +21,57 @@ Before you install the MongoDB Community Kubernetes Operator, you must:
    ```
    git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
    ```
+4. Review the possible Operator [deployment scopes](#understand-mongodb-community-operator-deployment-scopes) and configure the Operator to watch other namespaces, if necessary.
+
+### Understand Deployment Scopes
+
+You can deploy the MongoDB Community Kubernetes Operator with different scopes based on where you want to deploy Ops Manager and MongoDB Kubernetes resources:
```

Comment:
The Community Operator doesn't support Ops Manager :)

Reply:
+1

Reply:
oof. copy / paste mistake from the enterprise operator. fixed.